public_suffix_list.dat

```diff
@@ -13404,6 +13404,10 @@ githubusercontent.com
 githubpreview.dev
 github.io
 
+// Cloudbees, Inc.
+// Submitted by Mohideen Shajith <mshajith@cloudbees.com>
```

Comment:
Please use a non-personal email address.

Reply:
this is company email. Not my personal email-id

Reply:
What I meant is that the email address should be for a group or team rather than an individual. Even though it is a company domain email, it is still associated with a specific person. For example, if `Mohideen Shajith <mshajith@cloudbees.com>` leaves the company, we will not be able to reach your organization because this email will no longer be valid. I assume `mshajith` is tied to a specific person's name.